_data/navigation.yml

```diff
@@ -44,81 +37,16 @@
   - title: "Online Workshop Recommendations"
     url: "https://carpentries.org/online-workshop-recommendations/"
 
-- title: Learn
-  url: "/learn/"
-  side: left
-  dropdown:
-  - title: "Our Workshops"
-    url: "/workshops/"
-  - title: "Community Lessons"
-    url: "/community-lessons/"
-
-  # - title: "Our Curricula"
-  #   url: "/workshops-curricula/"
-  # - title: "Upcoming Workshops"
-  #   url: "/upcoming_workshops/"
-  # - title: "Past Workshops"
-  #   url: "/past_workshops/"
-  # - title: "Our Impact"
-  #   url: "/assessment/"
-
-# - title: Join Us
-#   url: "/volunteer/"
-#   side: left
-#   dropdown:
-#   - title: "Join Us"
-#     url: "/volunteer/"
-#   - title: "Get Involved"
-#     url: "/community/"
-#   - title: "Help Develop Lessons"
-#     url: "/involved-lessons/"
-#   - title: "Become a Member Organisation"
-#     url: "/membership/"
-#  - title: "Job Vacancies"
-#     url: "/jobs/"
-
-- title: Our community
-  url: "/maintainers/"
-  side: left
-  dropdown:
-#   - title: "The Carpentries Team"
-#     url: "https://carpentries.org/team/"
-  - title: "Our Maintainers"
-    url: "/maintainers/"
-#   - title: "Our Mentors"
-#     url: "/mentors/"
-#   - title: "Our Regional Coordinators"
-#     url: "/regionalcoordinators/"
-#   - title: "Our Trainers"
-#     url: "/trainers/"
-  - title: "Task Forces"
-    url: "/task-forces/"
-#   - title: "Current Member Organisations"
-#     url: "/members/"
-  - title: "The Carpentries Instructors"
-    url: "https://carpentries.org/instructors/"
-
-
 - title: Connect
   url: "/connect/"
   side: left
   dropdown:
   - title: "Blog"
     url: "/blog/"
-#   - title: "Community Calendar"
```

Comment:
Also here, I think some of these could be valuable breadcrumbs in the future (certainly calendar and perhaps community discussions)

Reply:
Rule 7: When someone starts a Blog or a Community Calendar, we can write new pages to support them or pull down the latest versions from upstream.

Reply:
I feel you are trivialising this a little, these comments do no harm except point you in the right direction.

Reply:
Calendar embedded by 2d7d0706.

Reply:
Restored several in 62b97559.